client/main/results.js

```diff
@@ -43,6 +43,10 @@ var ResultsView = Backbone.View.extend({
             this.$welcome.addClass('silky-welcome-panel-hidden');
         });
 
+        this.model.analyses().once('analysisResultsChanged', (event) => {
+            this.$welcome.addClass('silky-welcome-panel-hidden');
+        });
+
```

Comment:
do you want to remove the former event listener?
do you want to prevent the welcome from being inserted if analyses are going to be? (prevents flicker)(use a timer).